.gitignore

```diff
@@ -58,4 +58,5 @@ props-table
 **/__snapshots__/screen
 **/__snapshots__/diff
 errorScreenshots
+.vscode/launch.json
```

Comment:
If we're going to ignore this file we might as well ignore the whole .vscode folder instead